TOC/Proposal/voting-rules.md

```diff
@@ -0,0 +1,23 @@
+# Voting Rules for TOC Members
+
+This guideline is used for the decision-making process of TOC members in the TiDB community. Refer to Apache's voting rules, decisions are usually made by [lazy consensus](https://www.apache.org/foundation/glossary.html#LazyConsensus). If no consensus can be reached, the matter may be resolved by a majority vote. And potentially highly-controversial matters are recommended to be resolved by a majority vote, too.
```

Comment:
We should inline our own lazy consensus instead of link to how Apache does. We're not Apache, and we may make some changes based on that as well as not be affected if the Apache glossary changes.

Reply:
OK I see the lazy consensus definition below, we should link there instead of link to Apache page here.

Reply:
Thanks, I will link to the definition below.